src/mcts/search.cc

```diff
@@ -1009,7 +1010,12 @@ void SearchWorker::FetchSingleNodeResult(NodeToProcess* node_to_process,
   }
   // For NN results, we need to populate policy as well as value.
   // First the value...
-  node_to_process->v = -computation_->GetQVal(idx_in_computation);
+  auto board = search_->played_history_.Last().GetBoard(); 
+  //assuming leela is searching from root
+  auto contempt = 0.05;
+  if(node_to_process->depth % 2 == 0)
+    contempt = -contempt;
+  node_to_process->v = -computation_->GetQVal(idx_in_computation) + contempt;
```

Comment:
should contempt be a saturating operation - not result in a v outside of the range +/-1 ?

Reply:
I don't think it should matter much, except for interactions with EGTB, in which case its probably good for a checkmate to score higher than a DTZ. But it needs testing both ways probably. I think the level of contempt and some sort of dynamic contempt are going to be much bigger deals.

Reply:
It should be tested to make sure it will still take a mate instead of going for the mate in 1 as well...

Reply:
> should contempt be a saturating operation - not result in a v outside of the range +/-1 ?

we can keep V inside the range with an IF THEN statement checking the V value after the computation.